### PR TITLE
Quick pass on the README.

### PR DESCRIPTION
**Pull request by thruflo**
Only a minor pass pending a proper refresh and some better content. Basically added some badges and videos and a pass on the description etc.

Ideally this should have some component diagrammes and summary code but let's hold that until the v2. It looks like this: https://github.com/electric-sql/electric/tree/thruflo/readme-pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,90 @@
...
+</p>
 
-# ElectricSQL
+<p align="center">
+  <a href="https://github.com/electric-sql/electric/stargazers/"><img src="https://img.shields.io/github/stars/electric-sql/electric?style=social&label=Star&maxAge=2592000" /></a>
+  <a href="https://github.com/electric-sql/electric/actions"><img src="https://github.com/electric-sql/electric/workflows/CI/badge.svg" alt="CI"></a>
+  <a href="/Users/thruflo/Development/electric-sql/electric/LICENSE"><img src="https://img.shields.io/badge/license-Apache_2.0-green" alt="License - Apache 2.0"></a>
+  <a href="https://electric-sql.com/docs/reference/limitations"><img src="https://img.shields.io/badge/status-alpha-orange" alt="Status - Alpha"></a>
+  <a href="https://discord.gg/B7kHGwDcbj"><img src="https://img.shields.io/discord/933657521581858818?color=5969EA&label=discord" alt="Chat - Discord"></a>
+  <a href="https://twitter.com/nestframework" target="_blank"><img src="https://img.shields.io/twitter/follow/nestframework.svg?style=social&label=Follow @ElectricSQL"></a>
```
Comment (alco) [inline README.md:27]:
```suggestion
  <a href="https://twitter.com/ElectricSQL" target="_blank"><img src="https://img.shields.io/twitter/follow/ElectricSQL.svg?style=social&label=Follow @ElectricSQL"></a>
```

File: README.md [hunk 2]
```diff
@@ -1,46 +1,90 @@
...
+  <a href="https://twitter.com/nestframework" target="_blank"><img src="https://img.shields.io/mastodon/follow/109599644322136925.svg?domain=https%3A%2F%2Ffosstodon.org"></a>
```
Comment (alco) [inline README.md:28]:
```suggestion
  <a href="https://fosstodon.org/@electric" target="_blank"><img src="https://img.shields.io/mastodon/follow/109599644322136925.svg?domain=https%3A%2F%2Ffosstodon.org"></a>
```

File: README.md [hunk 3]
```diff
@@ -1,46 +1,90 @@
...
+
+- [Marc Shapiro](https://lip6.fr/Marc.Shapiro) and [Nuno Preguiça](https://asc.di.fct.unl.pt/~nmp), the co-inventors of CRDTs
+- [@bieniusa](https://linkedin.com/in/annette-bieniusa-b0807b145), the lead developer of [AntidoteDB](https://www.antidotedb.eu)
+- [@josevalim](https://www.linkedin.com/in/josevalim), the creator of the [Elixir](https://elixir-lang.org) programming language
+
+See the [Team](https://electric-sql.com/about/team) and [Literature](https://electric-sql.com/docs/reference/literature) pages for more details.
+
+# Contributing
 
 See the [Community Guidelines](https://github.com/electric-sql/meta) including the [Guide to Contributing](https://github.com/electric-sql/meta/blob/main/CONTRIBUTING.md) and [Contributor License Agreement](https://github.com/electric-sql/meta/blob/main/CLA.md).
```
Comment (alco) [inline README.md:84]: We also have CONTRIBUTING.md and CLA.md files in this repo. FYI I fixed a typo in the former some time ago. Wasn't aware we had the same file in the meta repo which has the same typo.

**Review by alco — APPROVED** [review]
The content looks great :+1: There's just a few typos in link URLs.

**Comment by thruflo** [comment]
Thanks for picking up all these, clearly wasn't concentrating properly.

**Pull request merged**